lib/wasi-types/src/wasi/bindings.rs

```diff
@@ -257,6 +257,8 @@ pub enum Errno {
     Memviolation,
     #[doc = " An unknown error has occured"]
     Unknown,
+    #[doc = " Invocation is pending until the waker is woken."]
+    Pending,
```

Comment:
Could this be represented by `::Again` ?

Reply:
It's leftovers, ill remove it